src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -3382,13 +3387,24 @@ private function check_for_page_caching() {
 			$headers['Authorization'] = 'Basic ' . base64_encode( wp_unslash( $_SERVER['PHP_AUTH_USER'] ) . ':' . wp_unslash( $_SERVER['PHP_AUTH_PW'] ) );
 		}
 
+		/**
+		 * Filters the URL used for checking page caching status in the Site Health tool.
+		 *
+		 * This filter allows customization of the URL used for determining if persistent
+		 * page caching is enabled.
+		 *
+		 * @since 6.6.3
+		 *
+		 * @param string $url The URL used for the caching check. Default is the home URL.
+		 */
+		$caching_url                   = apply_filters( 'site_status_persistent_page_cache_url', home_url( '/' ) );
```

Comment:
It would make sense to move this to a separate function and also call this function from `get_test_page_cache`. This way if there are any changes to the filter, they need to be done only in 1 place. `get_persistent_page_cache_url` could be a suitable name.